public/app/features/dashboard/components/SubMenu/SubMenu.tsx

```diff
@@ -45,9 +46,17 @@ class SubMenuUnConnected extends PureComponent<Props> {
       return null;
     }
 
+    const styles = css`
```

Comment:
Should this be memoized?

Reply:
To be honest, I think it's better if we move it to a separate function outside of component.